packages/state-transition/src/block/externalData.ts

```diff
@@ -13,13 +13,14 @@ export enum ExecutionPayloadStatus {
   valid = "valid",
 }
 
-export enum DataAvailableStatus {
-  preDeneb = "preDeneb",
-  notAvailable = "notAvailable",
+export enum DataAvailabilityStatus {
+  preData = "preData",
+  /* validator activities can't be performed on out of range data */
+  outOfRange = "outOfRange",
   available = "available",
 }
```

Comment:
We were using the correct casing for enum values at `packages/fork-choice/src/protoArray/interface.ts` 

```ts
export enum DataAvailabilityStatus {
  PreData = "PreData",
  /* validator activities can't be performed on out of range data */
  OutOfRange = "OutOfRange",
  Available = "Available",
}
```

Why moving it to other package changed the type case for it? All enum values should be PascalCase.